src/app/components/reportesDiarios/encabezadoReportesDiarios.component.ts

```diff
@@ -0,0 +1,312 @@
+import { Plex } from '@andes/plex';
+import { Router } from '@angular/router';
+import { Component, OnInit, HostBinding, Output, EventEmitter } from '@angular/core';
+// import { FormBuilder, FormGroup, Validators, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { Server } from '@andes/shared';
+import { Auth } from '@andes/auth';
+// import * as moment from 'moment';
+import { OrganizacionService } from '../../services/organizacion.service';
+import { AgendaService } from '../../services/turnos/agenda.service';
+import { TipoPrestacionService } from '../../services/tipoPrestacion.service';
+
+@Component({
+    selector: 'encabezadoReportesDiarios',
+    templateUrl: 'encabezadoReportesDiarios.html',
+
+})
+export class EncabezadoReportesDiariosComponent implements OnInit {
+    @HostBinding('class.plex-layout') layout = true; // Permite el uso de flex-box en el componente
+
+    // Variables comunes a varios reportes
+    // public disabled = true;
+    public showBotonImprimir = false;
+    public opcionesOrganizacion: any = [];
+    public opcionesReportes: any = [];
+    public opcionesPrestacion: any = [];
+    public parametros = {};
+    public organizacion;
+    public tipoReportes;
+    public prestacion;
+
+    public reporte: any = [];
+
+    // Variables "ResumenDiarioMensual"
+    public showResumenDiarioMensual = false;
+    public opcionesMes: any = [];
+    public opcionesAnio: any = [];
+    public anio;
+    public mes;
+
+    // Variables "PlanillaC1"
+    public showPlanillaC1 = false;
+    public fecha: any;
+
+    // Eventos
+    @Output() selected: EventEmitter<any> = new EventEmitter<any>();
+
+    constructor(
+        private plex: Plex,
+        private router: Router,
+        private server: Server,
+        private agendaService: AgendaService,
+        private auth: Auth,
+        private servicioOrganizacion: OrganizacionService,
+        public servicioPrestacion: TipoPrestacionService
+    ) {
+
+    }
+
+    public ngOnInit() {
+        // debugger
+
+        this.organizacion = null;
+        this.tipoReportes = null;
+        this.mes = null;
+        this.anio = null;
+
+        this.opcionesReportes = [
+            {
+                id: 1,
+                nombre: 'ResumenDiarioMensual'
```

Comment:
```suggestion
                nombre: 'Resumen diario mensual'
```
Para que se vea bien en el combo box. Hacer lo mismo con la otra opción y actualizar los strings en el generar reporte () para que entre al if